sdk/identity/identity/test/internal/node/interactiveBrowserCredential.spec.ts

```diff
@@ -12,8 +12,6 @@ import { InteractiveBrowserCredential } from "../../../src";
 import { MsalTestCleanup, msalNodeTestSetup } from "../../msalTestUtils";
 import { interactiveBrowserMockable } from "../../../src/msal/nodeFlows/msalOpenBrowser";
 
-import { URL } from "url";
```

Comment:
I was worried you were removing this from the runtime code, but it's just tests!